tests/jacoco-coverage-ratchet-test/Makefile

```
@@ -0,0 +1,28 @@
+PROGRAM=../../jacoco-coverage-rachet
+
+test: test12 test21 test13 test31
```

checkmake (0.2.2)

[warning] Target "test" should be declared PHONY.

(phonydeclared)